pkg/deployable/destroy.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2024 The Okteto Authors
```

Comment:
Common logic used by the local destroy command and the  remote-run destroy to executed the custom commands of deploy operations